GiniVision/Classes/ReviewViewController.swift

```diff
@@ -136,6 +156,24 @@ public typealias ReviewErrorBlock = (_ error: ReviewError) -> ()
     }
     
     /**
+     Convenience initializer for the `ReviewViewController` which allows to set a success block and an error block which will be executed accordingly.
+     
+     - parameter imageData:  JPEG representation as a result from the camera.
+     - parameter success:    Success block to be executed when image was rotated.
+     - parameter failure:    Error block to be executed if an error occured.
+     
+     - returns: A view controller instance allowing the user to review a picture of a document.
+     */
+    
+    @nonobjc
+    @available(*, deprecated)
+    public convenience init(_ imageData:Data, success: @escaping ReviewSuccessBlock, failure: @escaping ReviewErrorBlock) {
+        self.init(GiniImageDocument(data: imageData), successBlock: { (document,_) in
+            success(document.data)
+        }, failureBlock: failure as! GVCameraScreenFailureBlock)
```

Comment:
You should wrap it instead of casting it.

Reply:
Yes, actually there is a bug here. Thanks! 👍